tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -320,10 +320,26 @@ class Quint(moduleData: QuintOutput) {
         case "replaceAt" => quintArgs => ternaryApp(opName, tla.except)(incrIndexAtNthArg(1, quintArgs))
         case "slice" =>
           quintArgs => ternaryApp(opName, tla.subseq)(incrIndexAtNthArg(1, incrIndexAtNthArg(2, quintArgs)))
+        case "select" =>
+          quintArgs =>
+            binaryApp(opName,
+                (seq, test) => {
+                  val seqType = Quint.typeToTlaType(types(quintArgs(0).id).typ)
+                  val elemType = seqType match {
+                    case SeqT1(elem) => elem
+                    case invalidType => throw new QuintIRParseError(s"sequence ${seq} has invalid type ${invalidType}")
+                  }
+                  val elemParam = tla.param("__e", elemType)
+                  val resultParam = tla.param("__res", seqType)
+                  val elem = tla.name(elemParam._1.name, elemParam._2)
+                  val result = tla.name(resultParam._1.name, resultParam._2)
```

Comment:
This feels a bit cumbersome.
@Kukovec is there a more idiomatic way of writing this in the typed builder?

Reply:
Not currently, beyond
```suggestion
                  val eName = "__e"
                  val resName = "__res"
                  val elemParam = tla.param(eName, elemType)
                  val resultParam = tla.param(resName, seqType)
                  val elem = tla.name(eName, elemType)
                  val result = tla.name(resName, seqType)
```

Reply:
Though I don't see a reason not to add something like `tla.param(...).toName` if this feels too clunky as is.